master/internal/api_experiment.go

```diff
@@ -1469,6 +1469,21 @@ func (a *apiServer) parseAndMergeContinueConfig(expID int, overrideConfig string
 			fmt.Sprintf("override config must have single searcher type got '%s' instead", overrideName))
 	}
 
+	ctx := context.TODO()
+	w, err := workspace.WorkspaceByName(ctx, activeConfig.Workspace())
+	if err != nil && err != sql.ErrNoRows && err != db.ErrNotFound {
```

Comment:
This is a great point, we shouldn't be, I changed this!
The motivation behind this was that `activeConfig.Workspace()` is `""` when the experiment is created in the `Uncategorized` workspace (and hasn't been moved). But it doesn't make sense to do all of these error checks to avoid that case, and we shouldn't be avoiding handling that case. 
I refactored this section of the code to reassign the workspace to `Uncategorized` if `activeConfig.Workspace == ""`
Great catch!